### PR TITLE
Refs #11998 - support plugin httpd files without manage_httpd

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -265,6 +269,7 @@
   $manage_broker             = $pulp::params::manage_broker,
   $manage_db                 = $pulp::params::manage_db,
   $manage_httpd              = $pulp::params::manage_httpd,
+  $manage_plugins_httpd      = $pulp::params::manage_plugins_httpd,
```
Comment (ehelms) [inline manifests/init.pp]: Can you add a validate_bool for this?

Comment (jlsherrill) [inline manifests/init.pp]: oh yeah, meant to and forgot :+1:

Comment (jlsherrill) [inline manifests/init.pp]: @ehelms updated

**Comment by ehelms** [comment]
APT

**Pull request merged**